spacy/scorer.py

```diff
@@ -305,6 +305,8 @@ def score_spans(
         *,
         getter: Callable[[Doc, str], Iterable[Span]] = getattr,
         has_annotation: Optional[Callable[[Doc], bool]] = None,
+        consider_label: bool = True,
```

Comment:
I think `include_label` would be a clearer name.

Reply:
You know I'm notoriously bad at naming things, so happy to take your advice ;-)

Reply:
@adrianeboyd @honnibal `include_label`, or `include_labels` ?

Reply:
I think since it's `span.label` that `include_label` is better. Maybe Matt has a better suggestion than `include`, but it's also exactly what you wrote in the description, so?

Reply:
How about `labelled`?